images/router/haproxy/conf/haproxy-config.template

```diff
@@ -214,7 +214,7 @@ backend be_edge_http_{{$cfgIdx}}
   {{ end }}
   http-request set-header Forwarded for=%[src];host=%[req.hdr(host)];proto=%[req.hdr(X-Forwarded-Proto)]
                 {{ range $idx, $endpoint := endpointsForAlias $cfg $serviceUnit }}
-  server {{$endpoint.ID}} {{$endpoint.IP}}:{{$endpoint.Port}} check inter 5000ms cookie {{$endpoint.ID}}
+  server {{$endpoint.ID}} {{$endpoint.IP}}:{{$endpoint.Port}} check inter 5000ms cookie {{$endpoint.IDhash}}
```

Comment:
Needs to be `server {{$endpoint.IDHash}} {{$endpoint.IP}} ... `
The server `id`  is set to `{{$endpoint.ID}}` - it should match the IDHash - that value from the cookie is used to pick a specific server.  

Also `IDHash`  seems more consistent as a name than `IDhash`.